www_src/pages/login/form-input.jsx

```diff
@@ -3,13 +3,21 @@ var React = require('react');
 var FormInput = React.createClass({
   getDefaultProps: function () {
     return {
-      type: 'text'
+      type: 'text',
+      tabIndex: 0
     };
   },
+  checkForReturn: function (e) {
+    if (e.keyCode === 13) {
+      if (typeof this.props.onReturn === 'function') {
```

Comment:
Because it is a prop set by the parent, it could be `undefined` or just about anything another dev specifies.